Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/CiviMed.yml

```diff
@@ -0,0 +1,12 @@
+- type: vendingMachineInventory
+  id: CiviMedVendInventory
+  startingInventory:
+    Brutepack: 20
+    Ointment: 20
+    Bloodpack: 20
+    Gauze: 20
+    CigPackNFMedical: 10
```

Comment:
I'd suggest adding `Tourniquet` to the list: it's next to useless as a medical item (stops bleeding, but deals 5 blunt and 5 asphyxiation damage), however might be good for RP reasons.

Reply:
It's not a request, just a suggestion.
(Also I doubt that tourniquets will ever become something better than significantly worse version of gauze.)